Sources/SwiftDoc/Symbol.swift

```diff
@@ -58,7 +58,26 @@ public final class Symbol {
 
         return false
     }
-
+    
+    public var isInternal: Bool {
```

Comment:
I think either the implementation or the property name is highly misleading. With this implementation, a public property would return `true`. Same for an `open` or `fileprivate` property.

Also, the actual decision whether a symbol is internal is way more complicated. There are quite some cases to consider, especially with extensions. You can take a look at the existing implementation of `isPublic` where these cases are considered.

Reply:
.@Lukas-Stuehrk is correct. Determining access levels for symbols is extremely complicated in Swift. I've had to fix the implementation for `isPublic` a few times already to accommodate unforeseen edge cases (e.g. cases of public enumerations are always public, even though they never have access modifiers).